.github/workflows/pull_request.yml

```diff
@@ -1308,6 +1308,40 @@ jobs:
           docker ps --quiet | xargs --no-run-if-empty docker kill ||:
           docker ps --all --quiet | xargs --no-run-if-empty docker rm -f ||:
           sudo rm -fr "$TEMP_PATH"
+  FunctionalStatelessTestReleaseAnalyzer:
```

Comment:
Probably at least add another run with sanitizer (ASan)? Or just just replace this Release run with ASan?

Reply:
I want to start just with one because it requires suppressing errors for some tests.